sdk/cosmos/azure_data_cosmos/Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository.workspace = true
 rust-version.workspace = true
 homepage = "https://github.com/azure/azure-sdk-for-rust"
 documentation = "https://docs.rs/azure_data_cosmos"
-keywords = ["sdk", "azure", "rest", "cloud", "cosmos", "database"]
+keywords = ["sdk", "azure", "rest", "cloud", "cosmos"]
```

Comment:
I'd recommend keeping "database" and drop "azure", which is in the crate name already and takes precedence in search. See #2198

Reply:
I absolutely agree in general, but would like to get this landed to unblock the release pipeline and then I'll open a bikeshed for us to refine these for 0.23.0